src/helpers.coffee

```diff
@@ -82,20 +82,26 @@ exports.some = Array::some ? (fn) ->
 # out all non-code blocks, producing a string of CoffeeScript code that can
 # be compiled “normally.”
 exports.invertLiterate = (code) ->
-  # Create a placeholder for tabs, that isn’t used anywhere in `code`, and then
-  # re-insert the tabs after code extraction.
-  generateRandomToken = ->
-    "#{Math.random() * Date.now()}"
-  while token is undefined or code.indexOf(token) isnt -1
-    token = generateRandomToken()
-
-  code = code.replace "\t", token
-  # Parse as markdown, discard everything except code blocks.
-  out = ""
-  for item in marked.lexer code, {}
-    out += "#{item.text}\n" if item.type is 'code'
-  # Put the tabs back in.
-  out.replace token, "\t"
+  # `marked` really doesn’t process tabs too well. All this code is headed
+  # into JavaScript anyway, so replace the tabs with two spaces to make
+  # `marked`’s job a little easier.
+  code = code.replace /^(\t)+/gm, (match) ->
```

Comment:
`code` here is the source code of an entire file. This function is intending to replace all leading tabs (between the start of any line and the first non-tab character) with 2 spaces × the number of leading tabs on that line. Does it not do that?